packages/moonstone/MoonstoneDecorator/fontGenerator.js

```diff
@@ -1,81 +1,42 @@
 /*
- * This module loads Moonstone specific fonts. It only includes one function,
- * {@link moonstone/MoonstoneDecorator.fontGenerator} and is not intended to be directly
- * included by external developers.
+ * This module loads Moonstone's locale specific fonts.
+ *
+ * _This is not intended to be directly included by external developers._ The purpose of this is to
+ * override the existing "Moonstone" font family with a new typeface, conditionally when the system
+ * locale matches the corrosponding locale for the font (defined in this component's code).
+ *
  */
 // eslint-disable-next-line no-var
-var {addLocalizedFont, generateFontRules, generateFontOverrideRules} = require('@enact/ui/internal/localized-fonts');
+let {addLocalizedFont, generateFontRules, generateFontOverrideRules} = require('@enact/ui/internal/localized-fonts');
```

Comment:
@JayCanuck I think my editor automatically converted this one. Do you think this will be ok? We have `const` mentioned and used below, so this seems like an ok alteration. Think it will impact precompiling/prerendering/etc?

Reply:
yea this seems fine :+1:

Reply:
I guess we should remove the eslint `no-var` line.

Reply:
@webOS101 You're right!